js/main.js

```diff
@@ -80,9 +55,8 @@ setInterval(function() {
   str +=		'<span>' + secs + ' Seconds</span>';
 
   $('.countdown .countdown__counter').html(str);
-}, 1000); // Set the time of the interval, in ms, so 1000 (1s)
```

Comment:
It's not our fault if someone doesnt know the MS -> S conversion rate. If anything its indicative that we have a magic number that should be purged.